code/game/objects/items/weapons/tools.dm

```diff
@@ -181,6 +181,16 @@
 		M.appearance_flags = RESET_COLOR
 		. += M
 
+/obj/item/weapon/screwdriver/get_belt_overlay()
+	if(random_color)
+		var/mutable_appearance/body = mutable_appearance('icons/obj/clothing/belt_overlays.dmi', "screwdriver")
+		var/mutable_appearance/head = mutable_appearance('icons/obj/clothing/belt_overlays.dmi', "screwdriver_head")
+		body.color = color
+		head.overlays += body
+		return head
+	else
+		return mutable_appearance('icons/obj/clothing/belt_overlays.dmi', item_state) //Item state, because some screwdrivers use different icon files

```

Comment:
The abductor screwdriver refers to `abductor.dmi` to get its icon state. It uses the `item_state` of a regular screwdriver, however - `screwdriver_nuke` - and so I refer to it there.

I can also change its icon state directly if that's more consistent.